projects/demo/src/modules/app/pages.ts

```diff
@@ -1032,9 +1032,15 @@ export const pages: TuiDocPages = [
     {
         section: 'Experimental',
         title: 'Card',
```

Comment:
nit: CardMedium ?

Reply:
good catch, I will fix it

Reply:
But both CardMedium and CardLarge are on that page